cmd/destroy/destroy.go

```diff
@@ -244,6 +245,16 @@ func (dc *destroyCommand) getDestroyer(opts *Options) (destroyInterface, error)
 
 		destroyImage := ""
 		if manifest.Destroy != nil {
+			if opts.Name == "" {
+				if manifest.Name == "" {
+					manifest.Name = dc.buildCtrl.name
```

Comment:
where does `dc.builderCtrl` get `name` ?

Reply:
In the initializer. in the `destroy.go`. The name is the inferred name (from the cmap or the folder name)